projects/uitdatabank/reference/entry.json

```diff
@@ -2152,6 +2152,76 @@
         ]
       }
     },
+    "/events/{eventId}/organizer/{organizerId}": {
+      "parameters": [
+        {
+          "$ref": "#/components/parameters/eventId"
+        },
+        {
+          "$ref": "#/components/parameters/organizerId"
+        }
+      ],
+      "delete": {
+        "summary": "organizer - delete",
+        "description": "Deletes the organizer of the event with the given `eventId`.",
+        "operationId": "events-organizer-delete",
+        "tags": [
+          "Events"
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content. The organizer has been deleted successfully."
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "$ref": "#/components/responses/NotFound"
+          }
+        },
+        "security": [
+          {
+            "USER_ACCESS_TOKEN": []
+          },
+          {
+            "CLIENT_ACCESS_TOKEN": []
+          }
+        ]
+      },
+      "put": {
+        "summary": "organizer - update",
+        "description": "Updates the organizer of the event with the given `eventId`. A list of organizers can be found using our [Organizers guide](/docs/uitdatabank/entry-api/organizers/finding-and-reusing-organizers).\n\n<!-- theme: info -->\n\n> An organizer is not required on an event, and it can only have one.",
```

Comment:
```suggestion
        "description": "Updates the organizer of the event with the given `eventId`. A list of organizers can be found using our [guide about finding existing organizers](/docs/uitdatabank/entry-api/organizers/finding-and-reusing-organizers).\n\n<!-- theme: info -->\n\n> An organizer is not required on an event, and it can only have one.",
```

(We will also add guides about creating organizers etc so there will be mutliple "Organizer" guides)

Also I think you can make the link relative to the file on the filesystem (instead of the Stoplight URL) like this:
```
../entry-api/organizers/finding-and-reusing-organizers.md
```
Stoplight should automatically change it to whatever URL they make of it. But try it out first of course.

Reply:
That works indeed. I'll use that from now on.